src/base/io/utilities/writeSBML.m

```diff
@@ -530,7 +530,7 @@
 %the model.
 objectiveSense = 'maximize';
 
-if isfield(model,'osense') && model.osense == 1
+if isfield(model,'osenseStr') && strcmpi(model.osenseStr,'min')
```

Comment:
This line will throw an error if `model.osenseStr` does not exist.
We should separate this line in 2.

Reply:
No it will not. 
Even Matlab will not further evaluate a logical `&` statement if the first part evaluates to 0.
i.e. if `isfield(model,'osenseStr')` return `0`, `strcmpi(model.osenseStr,'min')` is not evaluated.

Reply:
:+1: